docs/tutorials/advanced_example.rst

```diff
@@ -236,228 +178,17 @@ Acquisition, Initialization and other aliases
 
 Setting the parameter structure
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-::
-
-  struct Params {
-    struct bayes_opt_boptimizer : public defaults::bayes_opt_boptimizer {
-        BO_PARAM(double, noise, 0.0);
-    };
-    struct bayes_opt_bobase {
-        BO_PARAM(int, stats_enabled, true);
-    };
-    struct stop_maxiterations {
-        BO_PARAM(int, iterations, 100);
-    };
-    struct stop_mintolerance {
-        BO_PARAM(double, tolerance, -0.02);
-    };
-    struct acqui_ucb {
-        BO_PARAM(double, alpha, 0.4);
-    };
-    struct init_randomsampling {
-        BO_PARAM(int, samples, 10);
-    };
-    struct kernel_squared_exp_ard : public defaults::kernel_squared_exp_ard {
-    };
-    struct opt_rprop : public defaults::opt_rprop {
-    };
-    struct opt_parallelrepeater : public defaults::opt_parallelrepeater {
-    };
-    struct opt_cmaes {
-        BO_PARAM(int, restarts, 1);
-        BO_PARAM(int, max_fun_evals, -1);
-    };
-  };
+.. literalinclude:: ../../src/tutorials/advanced_example.cpp
+   :language: c++
+   :linenos:
+   :lines: 6-36
 
 Creating and running the Bayesian Optimizer
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In your main function, you need to have something like the following: ::
-
-  // includes
-  // parameter structure
-
-  int main(int argc, char** argv)
-  {
-    // aliases
-    bayes_opt::BOptimizer<Params, modelfun<gp_t>, acquifun<acqui_t>, acquiopt<acqui_opt_t>, initfun<init_t>, statsfun<stat_t>, stopcrit<stop_t>> boptimizer;
-    // Instantiate aggregator
-    DistanceToTarget<Params> aggregator({1.5, 1.5});
-    boptimizer.optimize(eval_func(), aggregator);
-    std::cout << "New target!" << std::endl;
-    // Adding new target
-    aggregator = DistanceToTarget<Params>({2, 1.5});
-    boptimizer.optimize(eval_func<Params>(), aggregator, false);
-    // rest of code
-  }
-
-
-Running the experiment
-^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The section "Running the experiment" is nowhere to be seen in the new 
version of the file and I don't see how the source code replaces it.

Le 14/06/2016 10:02, Konstantinos Chatzilygeroudis a écrit :
>
> In docs/tutorials/advanced_example.rst 
> <https://github.com/resibots/limbo/pull/120#discussion_r66924508>:
>
> > -  {
> > -    // aliases
> > -    bayes_opt::BOptimizer<Params, modelfun<gp_t>, acquifun<acqui_t>, acquiopt<acqui_opt_t>, initfun<init_t>, statsfun<stat_t>, stopcrit<stop_t>> boptimizer;
> > -    // Instantiate aggregator
> > -    DistanceToTarget<Params> aggregator({1.5, 1.5});
> > -    boptimizer.optimize(eval_func(), aggregator);
> > -    std::cout << "New target!" << std::endl;
> > -    // Adding new target
> > -    aggregator = DistanceToTarget<Params>({2, 1.5});
> > -    boptimizer.optimize(eval_func<Params>(), aggregator, false);
> > -    // rest of code
> > -  }
> > -
> > -
> > -Running the experiment
> > -^^^^^^^^^^^^^^^^^^^^^^^^^
>
> What do you mean "no"?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/resibots/limbo/pull/120/files/53230d627372f0801bda7fe839f8b08b055024e9#r66924508>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AMmndJwRJnNv9TeooT6S6DrwZ85FSg0wks5qLmAZgaJpZM4Iw0yg>.
>